File: Hiroshima.Admin.Project/src/app/shared/models/TravellerFeedback.ts
import { ITravellerFeedback } from "src/app/shared/interfaces/ITravellerFeedback";
/**
 * Traveller feedback module
 */
export class TravellerFeedback implements ITravellerFeedback {
    /**
     * Traveller primary key
     */
    Id?: number;
    /**
     * Traveller id
     */
    TravellerId: number;
    /**
     * Traveller name
     */
    FullName: string;
    /**
     * Traveller contact number
     */
    ContactNumber: string;
    /**
     * Feedback description added by user
     */
    FeedbackDescription: string;
    /**
     * Feedback type like booking or any other query
     */
    FeedbackType?: string;
    /**
     * Comment by admin
     */
    CommentedByAdmin?: string
}

File: Hiroshima.API.Project/Hiroshima.Maas.DL/Entities/TravellerFeedbackModel/TravellerFeedback.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Hiroshima.Maas.DL.Entities.TravellerFeedbackModel
{
    public class TravellerFeedback:BaseEntity
    {
        public int TravellerId { get; set; }
        public string FullName { get; set; }
        public string ContactNumber { get; set; }
        public string FeedbackDescription { get; set; }
        public string FeedbackType { get; set; }
        public string CommentedByAdmin { get; set; }
    }
}
